src/services/item/plugins/app/appData/index.ts

```diff
@@ -24,6 +27,7 @@ const appDataPlugin: FastifyPluginAsync = async (fastify) => {
     // origins from the publishers table an build a rule with that.
 
     fastify.register(appDataFilePlugin, { appDataService });
+    if (WEBSOCKETS_PLUGIN) fastify.register(appDataWsHooks, { appDataService });
```

Comment:
Add `{ ... }`

Reply:
Sorry, I don't get it? Can you give be an example?

Reply:
```suggestion
if (WEBSOCKETS_PLUGIN) {
  fastify.register(appDataWsHooks, { appDataService });
}
```